buildscripts/azure/azure-linux-macos.yml

```diff
@@ -55,3 +55,11 @@ jobs:
         export PATH=$HOME/miniconda3/bin:$PATH
         buildscripts/incremental/test.sh
       displayName: 'Test'
+
+    - script: |
```

Comment:
Perhaps add a comment here about why this is last (to stop the env under which testing takes place accidentally containing conda-forge packages)?